### DISCUSSION TITLE
下個蜘蛛人不再是人？迪士尼為蜘蛛人做了個機器特技演員

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#93] [General]
WDI 研發的特技機器人共經歷四代，最原型是稱為「磚塊」（the brick）的機器人，能控制重心，並能控制空中姿態及自動旋轉。
透過程式控制，在繩上搖擺時先屈膝加速，達一定角度後「鬆手」藉慣性擺向空中，同時伸展身體，並由「磚塊」做完美轉體，最後躺平落在網上，完成一整套「蜘蛛人」在大樓間穿梭的動作。
WDI 再把兩套棍子機器人組合做成類人樣，便成為原型機，在此基礎上最佳化等，最終成為 Stuntronics 類人型特技機器人。從棍子機器人到 Stuntronics，WDI 團隊接受《Wired》採訪時省略許多未講，並沒有多說團隊的努力，只說如何使機器人更像真的蜘蛛人，以及確保安全性。
Stuntronics 外殼是樹脂，3D 列印製作，即使減速網破掉，WDI 團隊也表示樹脂外殼能保護內部昂貴精密的主機板、處理器及線路。
https://technews.tw/2021/10/16/walt-disney-imagineering-stuntronics/

**Comment by Jie2222** [comment]
不知道的話根本看不出來是機器人，感覺身體很柔軟很協調，酷

**Comment by HSU-YU-JEN** [comment]
之前有看過類似的機器人影片，裡面提到最難設計的部分就是機器人的重心，這個機器人能控制自己在空中的姿態與自動旋轉真的很厲害~

**Comment by Isongzhe** [comment]
感覺這樣的技術也可以應用到很多地方，例如遊戲裡 需要真人做動態捕捉的部分

**Comment by YuHengChen54** [comment]
這個技術好酷，這樣以後做電影就更簡單了

**Comment by JAIMEHUNG** [comment]
感覺可以讓角色做出更高難度的動作，讓機器人嘗試人類做不到的幅度。

**Comment by 109302013evantsao** [comment]
最早之前的特技電影是本人親自上陣，再來是有專業特技演員再加上CGI，未來就只需要這類機器人+CGI，我們就能有大量好看的特技片可以看了
